SoftLayer/managers/block.py

```diff
@@ -348,6 +351,9 @@ def order_duplicate_volume(self, origin_volume_id, origin_snapshot_id=None,
         if origin_snapshot_id is not None:
             order['duplicateOriginSnapshotId'] = origin_snapshot_id
 
+        if dependent_duplicate:
+            order['isDependentDuplicateFlag'] = 1
```

Comment:
Would be simpler to just have `order['isDependentDuplicateFlag'] = dependent_duplicate` and remove the if statement

Reply:
as we discussed, this isn't possible